app/Http/Controllers/Backend/UserController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use App\Exceptions\UserAlreadyMutedException;
+use App\Exceptions\UserNotMutedException;
+use App\Http\Controllers\Controller;
+use App\Models\User;
+use App\Models\UserMute;
+use Exception;
+
+class UserController extends Controller
+{
+
+    /**
+     * @throws UserAlreadyMutedException
```

Comment:
parameter types and return value is already documented in the body. A description is unnecessary in my opinion, since the function is self-describing, so I would not add it either.

Reply:
It's not about how well the function explains itself. The parameters `User $user, User $userToBeMuted` should be in the phpdoc-comment so that every IDE can give the correct context for a method. PHPstorm has an easy generator for that.